root/etc/e-smith/templates/etc/shorewall/shorewall.conf/60options

```diff
@@ -87,7 +87,7 @@ MARK_IN_FORWARD_CHAIN=No
 
 MINIUPNPD=No
 
-MODULE_SUFFIX=ko.xz
+MODULE_SUFFIX="ko ko.xz"
```

Comment:
I'd like to reset this variable to an empty value, since the `man shorewall.conf` states:

>        MODULE_SUFFIX=["extension ..."]
>            The value of this option determines the possible file extensions of kernel modules. The default value is "ko ko.gz ko.xz o o.gz o.xz gz xz".

Reply:
@filippocarletti prefers to leave the default mush similar to EPEL